ONSdigital/ras-deploy
# Add smoke tests to preprod/prod

# Motivation and Context
Add simple smoke test to check we can get to respondent-home-ui in
preprod and prod

# What has changed
Add section in preprod and prod to run smoke tests after deploying and slacking if a failure occurs.

# How to test?
Check builds to see failing smoke test and passing smoke test
/teams/rmras/pipelines/respondent-home-ui/jobs/respondent-home-ui-preprod-deploy/builds/24
/teams/rmras/pipelines/respondent-home-ui/jobs/respondent-home-ui-preprod-deploy/builds/25

# Links
https://trello.com/c/SwkUMf9O/99-dev-task-7-ext-access-eq-social-survey-rh
https://github.com/ONSdigital/ras-deploy/pull/37/files
